molsim/classes.py

```diff
@@ -1221,6 +1225,11 @@ def _calc_tau(self):
 								self.source.dV*1000 * self.mol.q(self.source.Tex)
 							)
 					)
+		#Set the tau to 0 if it exceeds the tau_threshold so as to to not impact the fit (Samer)
+		self.spectrum.tau[self.spectrum.tau>=self.tau_threshold] = 0.
+		#Set the tau to 0 if it exceeds the eup_threshold (Samer)
+		self.spectrum.tau[self.eup<=self.eup_threshold] = 0.
```

Comment:
I don't think this is the right sign - should it not be `self.eup >= self.eup_threshold` if you want to set tau to zero if it exceeds the threshold?

Reply:
Generally we want to exclude the lower energy lines, as they will often have effects that we can't account for in simulations. My comment is misleading though

Reply:
Can you rewrite the comment to convey your intention a little bit better then? Just make a local change, commit, push and it should update here.

Reply:
Still not perfect, the comment doesn't describe what the code is doing outside of setting `tau` to zero. Explain a bit what the `eup` part is doing (e.g. what are you identifying to set to zero and _why_)